sql-plugin/src/main/311until340-non330db/scala/com/nvidia/spark/rapids/shims/GpuBroadcastJoinMeta.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2020-2023, NVIDIA CORPORATION.
```

Comment:
```suggestion
 * Copyright (c) 2023, NVIDIA CORPORATION.
```

Reply:
Oops, this file was supposed to be deleted. It will come back when BroadcastNestedLoopJoin is updated.